pyproject.toml

```diff
@@ -1312,6 +1313,7 @@ apache-airflow-kubernetes-tests = { workspace = true }
 apache-airflow-providers = { workspace = true }
 apache-aurflow-docker-stack = { workspace = true }
 apache-airflow-shared-timezones = { workspace = true }
+apache-airflow-shared-secrets_masker = { workspace = true }
```

Comment:
This should change now to `apache-airflow-shared-secrets-masker`, given https://github.com/apache/airflow/pull/54449/files#r2288588821 ?